event/listener.php

```diff
@@ -158,7 +158,7 @@ public function acp_login_notification($event)
 
 			$messenger = new \messenger(false);
 			$messenger->template('acp_login', 'en');
-			$messenger->to($this->config['board_contact'], $this->config['board_contact_name']);
+			$messenger->to($this->config['sec_contact']);
```

Comment:
What will happen when no email is set? Maybe use a ternary here and use board_contact if no sec_contact is set.